### PR TITLE
fix(wallet-ui): show rejected offers properly

### LINKED ISSUE #6319: Wallet UI did not provide feedback on rejected offer
Constructed an offer with a give that included more funds than I had (due to a bug that is being filed separately).  When offer was submitted, I signed in Keplr.  The offer shows rejected in the CLI, but I was never given feedback.  The offer card remained as pending in the UI.  Clicking "Exit" did not do anything.  On reload, the "Pending" offer was still there.

Offers should resolve when they are rejected and show as clearly finished in the UI.

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/agoric-sdk/issues/6319

### CODE REVIEW

File: packages/wallet/ui/src/components/Proposal.jsx [hunk 1]
```diff
@@ -180,6 +181,12 @@ const Proposal = ({ offer, purses }) => {
           <pre>{JSON.stringify(args, null, 2)}</pre>
         </div>
       )}
+      {error !== undefined && (
```
Comment (turadg) [inline packages/wallet/ui/src/components/Proposal.jsx]: why not show the error if it's ever a string or other truthy object?
```suggestion
      {error && (
```

File: packages/wallet/ui/src/util/WalletBackendAdapter.js [hunk 2]
```diff
@@ -202,8 +202,17 @@ export const makeWalletBridgeFromFollower = (
           const { status } = updateRecord;
           console.log('offerStatus', { status, offers });
           const oldOffer = offers[status.id];
-          if (
-            oldOffer &&
+          if (!oldOffer) {
+            break;
```
Comment (turadg) [inline packages/wallet/ui/src/util/WalletBackendAdapter.js:207]: here we have an offerStatus update for an unknown offer. I think we'd benefit from `console.warn` that this happened

**Review by turadg — APPROVED** [review]

File: packages/wallet/ui/src/util/WalletBackendAdapter.js (hunk 2 above)
Comment (samsiegart) [reply packages/wallet/ui/src/util/WalletBackendAdapter.js:207]: Done

File: packages/wallet/ui/src/components/Proposal.jsx (hunk 1 above)
Comment (samsiegart) [reply packages/wallet/ui/src/components/Proposal.jsx]: Good point, just copypasta, done.

**Review by turadg — APPROVED** [review]

**Pull request merged**